### PR TITLE
change in annotation map(gen_vimdoc.py)

### PR DESCRIPTION
**Pull request by bhaumikankan**
I think this will fix issue #15453

### CODE REVIEW

**Comment by vigoux** [comment]
Could you run `./scripts/gen_vimdoc.py` to se e how that affects docs?

File: scripts/gen_vimdoc.py [hunk 1]
```diff
@@ -239,6 +239,7 @@
 annotation_map = {
     'FUNC_API_FAST': '{fast}',
     'FUNC_API_CHECK_TEXTLOCK': 'not allowed when |textlock| is active',
+    'FUNC_API_LUA_ONLY' : 'only allowed for lua'
```
Comment (gpanders) [inline scripts/gen_vimdoc.py:242]:
```suggestion
    'FUNC_API_LUA_ONLY' : 'only allowed from Lua'
```

**Review by gpanders — CHANGES_REQUESTED** [review]

**Comment by vigoux** [comment]
Can you also push the doc changes so that we see how that looks ?

**Comment by bhaumikankan** [comment]
sir @vigoux  can you help me to run gen_vimdoc.py

File: scripts/gen_vimdoc.py [hunk 2]
```diff
@@ -239,6 +239,7 @@
 annotation_map = {
     'FUNC_API_FAST': '{fast}',
     'FUNC_API_CHECK_TEXTLOCK': 'not allowed when |textlock| is active',
+    'FUNC_API_LUA_ONLY' : 'only allowed from lua'
```
Comment (gpanders) [inline scripts/gen_vimdoc.py:242]: `Lua` is a proper noun and should be capitalized (I included that in my last suggestion).

```suggestion
    'FUNC_API_LUA_ONLY' : 'only allowed from Lua'
```

**Review by gpanders — CHANGES_REQUESTED** [review]

File: scripts/gen_vimdoc.py (hunk 2 above)
Comment (bhaumikankan) [reply scripts/gen_vimdoc.py:242]: @gpanders  can you help me to run this .py file

**Comment by vigoux** [comment]
You'll have to install `doxygen` on your computer first, then simply do `./scripts/gen_vimdoc.py`

**Comment by bhaumikankan** [comment]
@vigoux i install doxygen on my computer and run this but it just open my cmd for few seconds and then cmd promt gone . i dont know whats happening

**Comment by muniter** [comment]
After the command is done do `git status` and the documentation should now be updated. This changes should affect `/runtime/doc/api.txt` commit the file and push so it can be reviewed.

**Comment by bhaumikankan** [comment]
@muniter I did this and check it but that command doesn't effect api.txt . Therse no change 🤔

**Comment by gpanders** [comment]
Try it again. I just ran the script locally with your changes and it does update the `runtime/doc/api.txt` file.

Once you run the script, commit and push the changes to the api.txt file so we can review them.

**Comment by bhaumikankan** [comment]
msgpack module not foun i am getting this error

**Comment by justinmk** [comment]
> msgpack module not foun i am getting this error

    pip3 install msgpack

**Comment by bhaumikankan** [comment]
yes i did that

**Comment by bhaumikankan** [comment]
@justinmk  i did this and run ./scripts/gen_vimdoc.py but it just create the command promt for few seconds and then its goes away and i check git status but there's nothing change

**Comment by justinmk** [comment]
@bhaumikankan this is just python/pip usage. you need to be a bit more tenacious. you are asking too much of  PR reviewers.

**Comment by justinmk** [comment]
closing in favor of https://github.com/neovim/neovim/pull/13972

**Pull request closed without merging**